dev/devicelab/bin/tasks/plugin_lint_mac.dart

```diff
@@ -50,6 +50,8 @@ Future<void> main() async {
           environment: <String, String>{
             'LANG': 'en_US.UTF-8',
           },
+          // TODO(jmagman): Flutter cannot build against ARM simulators https://github.com/flutter/flutter/issues/64502
+          canFail: true,
```

Comment:
This sucks.
The plugin linter tests are going to hit this problem with Xcode 12, too.
\cc @cyanglaz

Reply:
I'm not familiar with the syntax. Doesn "canFail" make the analyzer pass even hitting this problem?

Reply:
Yes, it's running this check but not failing the test if the linter fails.  Flutter is missing an ARM simulator slice, so will fail on Xcode 12.